### DISCUSSION TITLE
usage and license

### DISCUSSION THREAD

**Discussion by Mhdaw** [coqui-ai/TTS#4056] [General]
Hello, I want to build a free, public(will be published on github) for a competition, And I want to use this for text to speech, since it is a free and open source, I'm ok with the Non-commercial license?

**Comment by eginhard** [comment]
The code is licensed under the [MPL 2.0](https://github.com/coqui-ai/TTS/blob/dev/LICENSE.txt), which also allows commercial use. If you want to use a pretrained model, you should check their individual licenses here: https://github.com/coqui-ai/TTS/blob/dev/TTS/.models.json
